### PR TITLE
ALE/added_requirements.txt

### PR DESCRIPTION
**Pull request by annalevchenko04**
I have added requirements.txt. In file, there are packages that are suitable for pylint checking (some of them are older versions due to incompatibility issues)

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 def main():
     """Run administrative tasks."""
+
```
Comment (Strexas) [inline manage.py:9]: Delete line between comment and code

File: .github/workflows/pylint.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+        pip install django
+    - name: Analysing the code with pylint
+      run: |
+        pylint $(git ls-files '*.py')
```
Comment (Strexas) [inline .github/workflows/pylint.yml:24]: workflow fails

**Review by Strexas — CHANGES_REQUESTED** [review]
Needs changes

File: manage.py (hunk 1 above)
Comment (annalevchenko04) [inline manage.py:9]:
```suggestion
```

Comment (annalevchenko04) [reply manage.py:9]:
```suggestion

```

Comment (Strexas) [reply manage.py:9]: Unresolved as line is not deleted

Comment (Strexas) [reply manage.py:9]: locally delete line and push changes to pull request

File: .github/workflows/pylint.yml (hunk 2 above)
Comment (Strexas) [inline .github/workflows/pylint.yml:24]: Rename PR according to Jira convention

File: manage.py (hunk 1 above)
Comment (Strexas) [reply manage.py:9]: Make changes in this branch

Comment (annalevchenko04) [reply manage.py:9]: image

Comment (Strexas) [reply manage.py:9]: Locally make changes, commit and push commit to this branch

Comment (annalevchenko04) [reply manage.py:9]: image
??

Comment (Strexas) [reply manage.py:9]: Give me output of
```
git status
git push
```

Comment (annalevchenko04) [reply manage.py:9]: image

Comment (annalevchenko04) [reply manage.py:9]: image

Comment (annalevchenko04) [reply manage.py:9]: image

Comment (Strexas) [reply manage.py:9]: Try again, if there still error, we will today resolve it together

**Pull request closed without merging**